numpy/core/fromnumeric.py

```diff
@@ -62,6 +60,28 @@ def _wrapfunc(obj, method, *args, **kwds):
         return _wrapit(obj, method, *args, **kwds)
 
 
+def _wrapreduction(obj, method, name, axis, dtype, out, **kwargs):
```

Comment:
`method` should be named `ufunc`, and `name` should be named `method`, for consistency with `_wrapfunc`